.semaphore/release.yml

```diff
@@ -0,0 +1,29 @@
+version: v1.0
+name: Paprika official release
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+blocks:
+  - name: "🍳 Release"
+    task:
+      secrets:
+        - name: github-ssh
+        - name: paprika-env
+      prologue:
+        commands:
+          - checkout
+          - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+          - chmod 0600 ~/keys/github.key
+          - ssh-add ~/keys/github.key
+          - git config user.name "Paprika Semaphore 2.0"
+          - git config user.email "allison.chen@wegalvanize.com"
```

Comment:
we should get a more generic email, only one person should have access to it, so we can increase security and delegate to the CI the responsibility to deploy and push to master/npm.

Reply:
🤦‍♀️ yes